pymc3/smc/smc.py

```diff
@@ -189,14 +189,17 @@ def sample_smc(
     elif kernel.lower() == "metropolis":
         likelihood_logp = logp_forw([model.datalogpt], variables, shared)
 
+    if parallel and cores > 1:
+        pool = mp.Pool(processes=cores)
+
     while beta < 1:
         if parallel and cores > 1:
-            pool = mp.Pool(processes=cores)
             results = pool.starmap(likelihood_logp, [(sample,) for sample in posterior])
         else:
             results = [likelihood_logp(sample) for sample in posterior]
         likelihoods = np.array(results).squeeze()
-        beta, old_beta, weights, sj = calc_beta(beta, likelihoods, threshold)
+        beta, old_beta, weights, sj, = calc_beta(beta, likelihoods, threshold)
```

Comment:
```suggestion
        beta, old_beta, weights, sj = calc_beta(beta, likelihoods, threshold)
```

Reply:
good catch! I was playing with `calc_beta`and asking for more inputs :-)